### DISCUSSION TITLE
Neden Deveb ?

### DISCUSSION THREAD

**Discussion by hasanyalsiz** [data-deveb/deveb.css#16] [Soru-Yanıt] [answered]
Ben neden bunu kullanmalıyım ? cezbedici yanı ne ?

**Comment by volkantash** [comment, marked as answer]
Selamlar. Bu soruya birkaç ayrı yanıt verebilirim.

1. deveb.css yalnızca html ve css üzerine kurulmuştur. Javascript kullanılmamıştır. Bu yüzden tasarımlar daha iyi işleyebilmektedir.
2. deveb.css ile döşenek bölmeleri (layout dividing) oluşturabilirsiniz. Bu konuda özgür olursunuz. 1/1 ile 100/100 arasında dilediğiniz değerleri yazarak genişlikleri tanımlayabilirsiniz. Böylece tasarımlarda belli kalıplara uymak bağlarından kurtulursunuz.
3. Esner (flex) özelliklerinden kolayca yararlanabilirsiniz.
4. `class=""` değeri her ölçeği etkileyecek biçimde ayarlanmış iken ekran yâni germe ölçülerine göre tasarlamak için md="", xl="" gibi öznitelikler kullanırsınız. Böylece başka araçlarda `class="text-left md-text-center"` gibi yazar iken deveb.css ile `class="text-left" md="text-center"` gibi yazabilirsiniz. 
5. Dilerseniz deveb.css'i Türkçe kullanabilirsiniz.
6. Gelecekte sunulacak sekme, sarkan seçek (dropdown menu) gibi özellikler ile jssiz bir tema tasarlayabilirsiniz.
7. deveb.css html üstünden işleyebilecek yapıdadır. Bir başkası yazdığınız html değerlerini derlemek kıskacına sokar iken öte yandan çok ünlü bir başka ürün tasarımcıyı belirlediği kurallara kıstırabilir. deveb.css'de ise bir değeri değiştirince dinamik olarak işlediğini görürsünüz. Böylece koşullara göre yalnızca html altında yazdığınız değerleri değiştiren bir iş yaptığınızda ikinci bir iş yapmaya gerek kalmadan tasarımda amaçladığınızı elde edebilirsiniz.

Daha başka özellikleri de yeri gelince anabiliriz.
